.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 **/*.rs.bk
-./*.gcode
+/*.gcode
```

Comment:
just curious, do you know if ./ and / equivalent in gitignore?

Reply:
Not for me, which is why I changed it. Maybe a Windows thing? The docs specify only the / case: https://git-scm.com/docs/gitignore#_pattern_format

Reply:
Could be. Guess you learn something new everyday haha